### PR TITLE
Typed and untyped properties on deletes

### PR DESCRIPTION
**Pull request by benjay10**
There are a lot of issues with Virtuoso and implicit string datatypes. Virtuoso does not treat untyped strings the same as type strings. On top of that, there is no convention as to when the datatype is to be supplied and when not to. This leads to situations where data is not properly removed from the graph. This PR hopes to resolve this a bit by always supplying the typed and untyped variant of the property to delete queries.

### CODE REVIEW

File: lib/storeToTriplestore.js [hunk 1]
```diff
@@ -178,28 +179,39 @@ export async function insertData(store, graph) {
...
     });
-
-    await mas.updateSudo(`
-      DELETE DATA {
-        GRAPH ${rst.termToString(graph)} {
-          ${triplesSparql.join('\n')}
-        }
-      }`);
+    if (triplesSparql.length)
+      await mas.updateSudo(`
```
Comment (cecemel) [inline lib/storeToTriplestore.js]: So... my  experience with virtuoso quirck: only delete one langtype triple per one graph. Sorry...

Comment (cecemel) [reply lib/storeToTriplestore.js]: unless the array is always length == 1

Comment (benjay10) [reply lib/storeToTriplestore.js]: It is. Look at [this block of code](https://github.com/lblod/worship-positions-graph-dispatcher-service-loket/blob/954eeea5bb5799edc5b29fce07cdac50046ee64d/lib/deltaProcessing.js#L522). Triples with `termType` `Literal` are filtered from the normal deletion, and are put in stores of size 1, to be deleted one by one in separate queries. I agree that I could have put both workaround in the same place, or structured the code a bit clearer on that front.

**Review by cecemel — APPROVED** [review]

File: lib/storeToTriplestore.js (hunk 1 above)
Comment (cecemel) [reply lib/storeToTriplestore.js]: check

**Pull request merged**